R/make_geojson.R

```diff
@@ -0,0 +1,37 @@
+#' Function to make geojson for tippecanoe (zones)
+#' @param z sf data frame with zones
+#' @param path folder to save results
+#' @example 
+# z_stats <- routes_to_zone_stats(r_fast, "fastest", "commute")
+# zones <- readRDS("../inputdata/iz_scotlands_uk.Rds")
```

Comment:
We will need to tidy this up and maybe create example datasets with 

```
usethis::use_data()
```

But this is fine for now.

Reply:
Wasn't sure this would be in the long-term workflow but needed for current vis